File: mobile-app/app/CinemaApplication/app/src/main/java/com/example/luulac/cinemaapplication/data/models/showtimes/ShowTimeListModel.java
package com.example.luulac.cinemaapplication.data.models.showtimes;

import java.util.List;

public class ShowTimeListModel {

    private String cinemaName;
    private String filmName;
    private String filmImg;
    private String cinemaGroupName;
    private String groupCinemaLogo;
    private List<ShowTimeChildModel> showTimeChildModels;

    public String getCinemaGroupName() {
        return cinemaGroupName;
    }

    public void setCinemaGroupName(String cinemaGroupName) {
        this.cinemaGroupName = cinemaGroupName;
    }

    public String getCinemaName() {
        return cinemaName;
    }

    public void setCinemaName(String cinemaName) {
        this.cinemaName = cinemaName;
    }

    public List<ShowTimeChildModel> getShowTimeChildModels() {
        return showTimeChildModels;
    }

    public void setShowTimeChildModels(List<ShowTimeChildModel> showTimeChildModels) {
        this.showTimeChildModels = showTimeChildModels;
    }

    public String getGroupCinemaLogo() {
        return groupCinemaLogo;
    }

    public void setGroupCinemaLogo(String groupCinemaLogo) {
        this.groupCinemaLogo = groupCinemaLogo;
    }

    public String getFilmName() {
        return filmName;
    }

    public void setFilmName(String filmName) {
        this.filmName = filmName;
    }

    public String getFilmImg() {
        return filmImg;
    }

    public void setFilmImg(String filmImg) {
        this.filmImg = filmImg;
    }
}

File: mobile-app/api/CinemaBookingSolution/CinemaBookingCore/Data/Models/ShowTimeListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CinemaBookingCore.Data.Models
{
    public class ShowTimeListModel
    {
        public String CinemaName { get; set; }
        public String CinemaGroupName { get; set; }
        public String FilmName { get; set; }
        public List<ShowTimeChildModel> ShowTimeChildModels { get; set; }
        public String GroupCinemaLogo { get; set; }
        public String FilmImg { get; set; }
    }
}
